frontend/app/components/work-packages/work-package-cache.service.ts

```diff
@@ -74,8 +83,10 @@ export class WorkPackageCacheService {
       state.clear();
     }
 
-    state.putFromPromiseIfPristine(
-      () => this.apiWorkPackages.loadWorkPackageById(workPackageId, forceUpdate));
+    if (workPackageId.toString() !== 'new') {
+    state.putFromPromiseIfPristine(() =>
+        this.apiWorkPackages.loadWorkPackageById(workPackageId, forceUpdate));
+    }
```

Comment:
The responsibilities between `WorkPackageCacheService` and `ApiWorkPackageService` are not well separated. For `list`, the schemas are loaded by the `WorkPackageCache` while the `ApiWorkPackageService` loads the schema if `loadWorkPackage` is called: https://github.com/opf/openproject/pull/5063/files#diff-030dc6d5af01418720e8f296bf3b048aR65